Cargo.toml

```diff
@@ -12,11 +12,13 @@ keywords = ["database", "postgres", "migration", "sql", "mysql"]
 [dependencies]
 regex = "0.1"
 url = "0.5"
-postgres = "0.11"
+postgres = { version = "0.11.3", features = ["openssl"] }
```

Comment:
keep 0.11 as version so we get the minor updates (postgres is currently 0.11.4 for example)